packages/api/app/Controllers/Http/TechnologyOrderController.js

```diff
@@ -1,10 +1,76 @@
+const Config = use('Adonis/Src/Config');
+const Technology = use('App/Models/Technology');
+const TechnologyOrder = use('App/Models/TechnologyOrder');
 const { orderStatuses, errorPayload, errors } = require('../../Utils');
+const Mail = require('../../Utils/mail');
 
 const Bull = use('Rocketseat/Bull');
 const Job = use('App/Jobs/SendMail');
 
-const TechnologyOrder = use('App/Models/TechnologyOrder');
 class TechnologyOrderController {
+	async technologyOrder({ params, request }) {
+		const technology = await Technology.findOrFail(params.id);
+		return TechnologyOrder.query()
+			.where('technology_id', technology.id)
+			.withFilters(request)
+			.withParams(request, { filterById: false });
+	}
+
+	async index({ request }) {
+		return TechnologyOrder.query()
+			.with('technology', (technology) =>
+				technology.select('id').with('users', (users) => users.select('id')),
+			)
```

Comment:
Se for sempre querer esse comportamento, pode adicionar o `withPivot` diretamente no model:

```js
technology() {
  return this.belongsToMany(/** ... */).withPivot(['id']);
}
```

E entao, utilizar somente desta forma:
```suggestion
			.with('technology')
```

Reply:
Essa query é porque preciso listar quem são os usuários responsáveis pela tecnologia e quem é o owner.
Só com o `.with('technology')` não vou ter a lista de usuários responsáveis pela tecnologia.